src/main/java/org/motechproject/mots/scheduler/ExampleJob.java

```diff
@@ -0,0 +1,59 @@
+package org.motechproject.mots.scheduler;
+
+import java.time.Instant;
+import java.util.Date;
+import lombok.extern.slf4j.Slf4j;
+import org.quartz.JobExecutionContext;
+import org.quartz.Trigger;
+import org.quartz.TriggerBuilder;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+@Slf4j
+@Component
+public class ExampleJob extends BaseJob {
+
+  public static final String NAME = "Example job";
+
+  private static final String DESCRIPTION = "This is a stub of a future task for sending messages";
+
+  @Value("${scheduled-tasks.example.interval}")
+  private int intervalInSeconds;
```

Comment:
we should be able to change this in the application (on frontend) not in the configs